spec/lib/dispatch-rider/subscriber_spec.rb

```diff
@@ -73,15 +73,15 @@ def process(options)
 
     # kills travis sometimes so leaving it here as tested documentation
     describe "process termination", if: false do
-      before { subject.demultiplexer.stub(:stop){ throw :got_stopped } }
+      before { allow(subject.demultiplexer).to receive(:stop) { throw :got_stopped } }
 
       context "when process quits" do
         before do
           stub_const("Quiter", Class.new(DispatchRider::Handlers::Base) {
-            def process(options)
-              Process.kill("QUIT", 0)
-            end
-          })
+                               def process(options)
```

Comment:
Unused method argument - `options`. If it's necessary, use `_` or `_options` as an argument name to indicate that it won't be used. You can also write as `process(*)` if you want the method to accept any arguments but don't care about them.